<?xml version="1.0"?>
<launch>
    <arg name="save_file" default="false" />
    <arg name="camera_ip" default="169.254.59.15" />      <!-- change to your camera ip -->
    <arg name="use_external_intri" default="false" />
    <arg name="fx" default="1543.49" />
    <arg name="fy" default="1543.49" />
    <arg name="u" default="798.715" />
    <arg name="v" default="763.939" />
    <arg name="tf_x" default="0" />
    <arg name="tf_y" default="0" />
    <arg name="tf_z" default="0" />
    <arg name="tf_qx" default="0" />
    <arg name="tf_qy" default="0" />
    <arg name="tf_qz" default="0" />
    <arg name="tf_qw" default="1" />

    <node name="mechmind_camera_start" pkg="mecheye_ros_interface" type="start" respawn="true" respawn_delay="1" output="screen">
        <param name="save_file" value="$(arg save_file)" />
        <param name="camera_ip" value="$(arg camera_ip)" />
        <param name="use_external_intri" value="$(arg use_external_intri)" />
        <param name="fx" value="$(arg fx)" />
        <param name="fy" value="$(arg fy)" />
        <param name="u" value="$(arg u)" />
        <param name="v" value="$(arg v)" />
        <param name="tf_x" value="$(arg tf_x)" />
        <param name="tf_y" value="$(arg tf_y)" />
        <param name="tf_z" value="$(arg tf_z)" />
        <param name="tf_qx" value="$(arg tf_qx)" />
        <param name="tf_qy" value="$(arg tf_qy)" />
        <param name="tf_qz" value="$(arg tf_qz)" />
        <param name="tf_qw" value="$(arg tf_qw)" />
    </node>
</launch>
